SocialCareCaseViewerApi/V1/Infrastructure/Worker.cs

```diff
@@ -30,7 +30,6 @@ public class Worker : IAuditEntity
         public string LastName { get; set; } = null!;
 
         [Column("is_active")]
-        [Required]
         public bool IsActive { get; set; }
```

Comment:
Should we add an update to the schema file to update all null is_active values to true i.e. 
`update dbo.sccv_worker set is_active = true where is_active isnull;`

Reply:
Implemented